### ISSUE TITLE
"webui\backend\operations_gguf.py:36: UserWarning: ..."

### ISSUE THREAD

**Issue by kyoushinosensei** [lllyasviel/stable-diffusion-webui-forge#2653]
Recently, I have been unable to recreate the image using the same prompt and seed, using two loras and the gguf q8 flux 1 dev model, I often see these errors in the console:

**webui\backend\operations_gguf.py:36: UserWarning: To copy construct from a tensor, it is recommended to use sourceTensor.clone().detach() or sourceTensor.clone().detach().requires_grad_(True), rather than torch.tensor(sourceTensor).
  return super().__new__(cls, torch.tensor(tensor.data), requires_grad=requires_grade)**

**ERROR lora diffusion_model.double_blocks.9.txt_mlp.2.weight**

**ERROR lora diffusion_model.single_blocks.37.linear2.weight**

RTX 2080 Ti (11 gb. VRAM), 32gb. RAM, Windows 10.

The problems started on the 11th, before that everything worked fine
